pkg/api/types.go

```diff
@@ -499,18 +499,16 @@ func (l *VolumeList) Set(value string) error {
 	if len(value) == 0 {
 		return fmt.Errorf("invalid format, must be source:destination")
 	}
-	mount := strings.Split(value, ":")
-	switch len(mount) {
-	case 1:
-		mount = append(mount, "")
-		fallthrough
-	case 2:
-		mount[0] = strings.Trim(mount[0], `"'`)
-		mount[1] = strings.Trim(mount[1], `"'`)
-	default:
-		return fmt.Errorf("invalid source:path definition")
+	var mount []string
+	pos := strings.LastIndex(value, ":")
+	if pos == -1 {
+		mount = []string{value, "."}
+	} else {
+		mount = []string{value[:pos], value[pos+1:]}
 	}
-	s := VolumeSpec{Source: filepath.Clean(mount[0]), Destination: filepath.Clean(mount[1])}
+	mount[0] = strings.Trim(mount[0], `"'`)
+	mount[1] = strings.Trim(mount[1], `"'`)
```

Comment:
Do we have unit tests for this?

Would be nice to explicitly test this with good and bad input.